tools/shoestring/shoestring/healthagents/websockets.py

```diff
@@ -5,13 +5,12 @@
 from zenlog import log
 
 from shoestring.internal.ConfigurationManager import parse_time_span
-from shoestring.internal.NodeFeatures import NodeFeatures
 
 NAME = 'REST websockets'
 
 
 def should_run(node_config):
-	return NodeFeatures.API in node_config.features
+	return node_config.full_api
```

Comment:
Since websocket is available, should this run on light?

Reply:
Sorry for confusing you. 
Light REST didn't have websocket support.

I discovered that my node has a websocket because it is enabled in the `rest-api-https-proxy` service.
I have fix in this commit [decac18](https://github.com/symbol/product/pull/1165/commits/decac186805d2ab5f288abfa65b657dc7b0d124c)